src/salesforce_uploader.js

```diff
@@ -40,29 +40,37 @@ type apiGatewayLambdaInput = {
 
 async function copyToUploadedFolder (stage, s3Path, sfFileName) {
   try {
-    let uploadedPath = `uploaded/${sfFileName}`
+    const uploadedPath = `uploaded/${sfFileName}`
     await
-      copyObject(s3Path, uploadedPath)
+    copyObject(s3Path, uploadedPath)
   } catch (err) {
     console.error('error copying fulfilment file to uploaded directory')
     console.log(err)
   }
 }
 
+/**
+ *  Example input:
+ *    Header: apiToken:*********
+ *    Body: { "date":"2020-01-10",  "amount":1}
+ *
+ *  The apiToken is specified in fulfilment.private.json under expectedToken.
+ *  FIXME: Why not just use API Gateway provided API Keys?
```

Comment:
This might be a repeat of the practice to work around Zuora's limitation of not being able to include custom headers in notification webhooks, and thus carried over and applied to Salesforce webhook receivers. Hence +1 to switching to use API Gateway keys.